### PR TITLE
Add webpack base configuration and extend it for dev and production

### PR DESCRIPTION
**Pull request by adrian-bobev**
Extract common configurations in a separate base file and extend it for production and development configurations.

### CODE REVIEW

**Comment by review-ninja** [comment]
<a href="https://app.review.ninja/LittleFurryBastards/report-it/pull/33" target="_blank">ReviewNinja

File: webpack.base.config.js [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+        test: /\.scss$/,
+        exclude: [FOLDER.nodeModules, path.join(__dirname, 'app/components')],
+        loader: ExtractTextPlugin.extract('style', 'css?modules&localIdentName=[name]__[local]!postcss!sass')
+      }
+    ]
+  },
+
+  postcss: function () {
+    return [
+      require('stylelint')(stylelintConfig),
```
Comment (Chariyski) [inline webpack.base.config.js]: This is not needed for production

File: webpack.base.config.js [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+
+  resolve: {
+    extensions: ['', '.js', '.jsx']
+  },
+
+  module: {
+    preLoaders: [
+      {
+        test: /\.(es6|jsx)$/,
+        loaders: ['eslint'],
```
Comment (Chariyski) [inline webpack.base.config.js]: ESlint is not needed for production

File: webpack.base.config.js (hunk 1 above)
Comment (adrian-bobev) [inline webpack.base.config.js]: !resolved

File: webpack.base.config.js (hunk 2 above)
Comment (adrian-bobev) [inline webpack.base.config.js]: !resolved

**Comment by Chariyski** [comment]
:+1:

**Pull request merged**